AirportTicketBooking/CSVIO/ReadFromCsvFile.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Linq;
+using AirportTicketBooking.Services;
+
+namespace AirportTicketBooking
+{
+    public class ReadFromCsvFile
+    {
+        public void RegisterFlightsData(string filePath, FileServices fileServices, Validation validation)
+        {
+            var fileLines = fileServices.ReadCsvFile(filePath);
+                var csvValidationErrors = validation.ValidateFlights(fileLines);
+
+                if (csvValidationErrors.Any())
+                {
+                    foreach (var error in csvValidationErrors)
+                    {
+                        Console.WriteLine(error);
+                        Environment.Exit(1);
```

Comment:
closing the console is not informative to the user about what happened and why the application closed, you should handle it in a way that is fault tolerant, so even when part of the application is not working the rest of it should work.

in your case here i would suggest sending a message to the user that there is an issue with the reading and cloud not serve your request, then return him to the main menu, to register the  errors usually on production you would have a log serves that register  the user requests but you dont have to do it here

Reply:
Refactored!!